lib/evss/base_service.rb

```diff
@@ -50,7 +50,7 @@ def vaafi_headers
         # Vary by user
         'va_eauth_firstName' => @user.first_name,
         'va_eauth_lastName' => @user.last_name,
-        'va_eauth_issueinstant' => @user.last_signed_in.iso8601,
+        'va_eauth_issueinstant' => Time.now.utc.iso8601,
```

Comment:
See my comment above. If we have to do it this way, okay, but it would be better if we didn't have to.